sycl/doc/extensions/LevelZeroBackend/LevelZeroBackend.md

```diff
@@ -7,8 +7,12 @@ It is built on top of Level-Zero runtime enabled with [Level-Zero API](https://s
 The Level-Zero backend is aimed to provide the best possible performance of SYCL application on a variety of targets supported.
 The currently supported targets are all Intel GPUs starting with Gen9.
 
-NOTE: This specification is a draft. While describing the currently implemented behaviors it is known to be not complete nor exhaustive.
-      We shall continue to add more information, e.g. explain general mapping of SYCL programming model to Level-Zero API.
+This extension provides a feature-test macro as described in the core SYCL specification section 6.3.3 "Feature test macros".
+Therefore, an implementation supporting this extension must predefine the macro `SYCL_EXT_ONEAPI_BACKEND_LEVEL_ZERO`.
+Applications can test for the existence of this macro to determine if the implementation supports this feature.

```

Comment:
Since this macro is a feature-test macro, its value should tell which revision of the extension the implementation supports.  This allows applications to test the value to see if an API is available.  We should use the same numbering scheme as other extensions, so this first revision will be revision `1`, the next will be `2`, etc.  I suggest using a table to document the revision numbers as we do in [device-if](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/DeviceIf/device_if.asciidoc) and other extensions.  For this first revision, the table will have just one row.

This is especially important now that we consider this extension "stable" (not "experimental").

Reply:
OK